cljfmt/src/cljfmt/core.cljc

```diff
@@ -699,16 +724,19 @@
   (and (or (z/list? zloc) (z/list? (z/up zloc)))
        (some #(matching-form-index? zloc % context) form-indexes)))
 
-(defn align-form-columns [form aligned-forms alias-map]
-  (let [ns-name  (find-namespace (z/of-node form))
-        context  {:alias-map alias-map, :ns-name ns-name}
+(defn align-form-columns [form aligned-forms alias-map & [{:keys [align-single-column-lines?]}]]
+  (let [ns-name (find-namespace (z/of-node form))
+        context {:alias-map alias-map, :ns-name ns-name}
         aligned? #(matching-form? % aligned-forms context)]
-    (transform form edit-all aligned? align-columns)))
+    (transform form edit-all aligned? #(align-columns % align-single-column-lines?))))
 
 (defn realign-form
   "Realign a rewrite-clj form such that the columns line up into columns."
-  [form]
-  (-> form z/of-node align-columns z/root))
+  [form & [{:keys [align-single-column-lines?]}]]
```

Comment:
This style of adding optional arguments should be avoided for two reasons:
1. It's less efficient than explicit arities
2. It doesn't error if too many arguments are given to the function.